tests/gui/test_02_import_data.py

```diff
@@ -129,33 +122,23 @@ def test_page_02_loads_maxquant_testfiles(mock_file_uploader: MagicMock):
 
     assert not at.exception
 
-    assert (
```

Comment:
I think it would eb important to test for each software, that the widgets are correctly initialized.

Reply:
yup, but e.g. the `loader` is not in the session stage at this stage